tests/opacity_test.py

```diff
@@ -1,26 +1,40 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
+import pytest
 import scipp as sc
 from scipp.testing import assert_identical
 
 from ess import polarization as pol
 
 
 def test_opacity_from_cell_params() -> None:
-    # In practice, the pressure and cell length are constant, this is just for testing.
+    # In practice, the cell pressure, length, and temperature are constant, this is
+    # just for testing.
     pressure = sc.array(dims=['pressure'], values=[1.0, 2.0], unit='bar')
-    cell_length = sc.array(dims=['cell_length'], values=[1.0, 2.0], unit='m')
+    length = sc.array(dims=['cell_length'], values=[1.0, 2.0], unit='m')
+    temperature = sc.array(dims=['temperature'], values=[200.0, 400.0], unit='K')
     wavelength = sc.array(dims=['wavelength'], values=[1.0, 2.0], unit='nm')
     opacity_function = pol.he3_opacity_from_cell_params(
-        pressure=pressure, cell_length=cell_length
+        pressure=pressure, length=length, temperature=temperature
     )
     opacity = opacity_function(wavelength)
     assert_identical(2 * opacity['pressure', 0], opacity['pressure', 1])
     assert_identical(2 * opacity['cell_length', 0], opacity['cell_length', 1])
     assert_identical(2 * opacity['wavelength', 0], opacity['wavelength', 1])
+    assert_identical(opacity['temperature', 0], 2 * opacity['temperature', 1])
     assert opacity.unit == ''
 
```

Comment:
This is testing if (i) together with wavelength in nm the opacity will have correct unit and (ii) test if outputs at two different values (in pressure/cell length/wavelength) is consistent?
Question: why the temperature 400K is taken *2, not 200K?

Reply:
As far as I can tell, the opacity should be proportional to pressure, length, and wavelength, but inversely-proportional to temperature? That is why the `2*` is for the higher temperature in this case. But please do not take my work, check the equations thoroughly!

Reply:
Oh right. I tried to more understand what the code is doing and then forgot about the inversed relation to "T" - my fault. All fits to the function of Opacity!